vpr/src/pack/cluster.cpp

```diff
@@ -1331,6 +1402,12 @@ static enum e_block_pack_status try_pack_molecule(t_cluster_placement_stats* clu
                         }
                     }
 
+                    //update cluster PartitionRegion if atom with floorplanning constraints was added
+                    if (cluster_pr_needs_update) {
```

Comment:
comment why you have a bool to update -> also change this to always return true and check that it still works, because it should